agent/engine/task_manager.go

```diff
@@ -53,6 +54,19 @@ const (
 	stoppedSentWaitInterval                  = 30 * time.Second
 	maxStoppedWaitTimes                      = 72 * time.Hour / stoppedSentWaitInterval
 	taskUnableToTransitionToStoppedReason    = "TaskStateError: Agent could not progress task's state to stopped"
+
+	fieldTaskArn       = "taskARN"
```

Comment:
nit: I dont mind much either way, but personally I don't like when constants are used as a string and named the same thing as the actual string. imo it's a little easier to read just using the string directly.

Reply:
I agree it reads better but it also opens the door for some people using `TaskARN`, `taskarn`, `ARN` etc. let me see if I can rename it so it's easier on the eye

Reply:
ended up just using the strings directly

Reply:
aaand.. decided to create a package to store the constants to use as fields. In my own tests I was being so inconsistent with the field names, so I think this is better if we are to parse those logs in an automated way in the future